manuscript/building/04_tidying_up.md

```diff
@@ -26,6 +26,9 @@ Next, you need to define a function to wrap the basic idea. You could use the pl
 ...
 const CleanWebpackPlugin = require("clean-webpack-plugin");
 
+// for clean-webpack-plugin@3.0.0, use named exported
+const { CleanWebpackPlugin } = require("clean-webpack-plugin"); 
```

Comment:
I would replace the entire line above. :)

I'll try to merge + publish over the weekend.

Reply:
cool! take you time.